spec/01-unit/22-grpc-utils_spec.lua

```diff
@@ -42,4 +42,9 @@ describe("grpc tools", function()
       "Added.Final",
     }, methods)
   end)
+
+  it("includes kong/include in the load path", function()
+    local grpc_tools_instance = grpc_tools.new()
+    assert.matches("kong/include$", grpc_tools_instance.protoc_instance.paths[3])
+  end)
```

Comment:
Add a test to load the .proto file with pure luarocks loader.
```suggestion
  end)
  it("loads .proto files with luarocks loader", function()
    local loader = require "luarocks.loader"
    local kong_init_file, _, _ = loader.which("kong")
    local kong_include_dir = kong_init_file:gsub("init%.lua$", "include")
    
    local protoc = require "protoc"
    local p = protoc.new()
    p.include_imports = true
    p:addpath(kong_include_dir)
    assert(p:loadfile("opentelemetry/proto/collector/trace/v1/trace_service.proto"))
  end)
```